src/interfaces.ts

```diff
@@ -0,0 +1,47 @@
+export interface Client {
```

Comment:
```suggestion
export interface User {
```
це швидше за все юзер, як бачимо це просто персональні дані, а от Client це вже підвид юзера в якого можуть бути нові поля, такі як корзина чи рахунок

Reply:
тоже самое